ci/buildspec_deploy.yml

```diff
@@ -49,7 +45,28 @@ phases:
       - aws ecs wait services-stable --cluster "${DEV_ECS_BACKEND_CLUSTER}" --services "${DEV_ECS_BACKEND_SERVICE}"
 
       # db:reseed
-      - docker run -v /tmp/secrets:/secrets --env PGPASSFILE=/secrets/pgpass "${ECR_ACCOUNT_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com/${ECR_NAMESPACE}/backend:migrate" sh -c "chown node:node /secrets && su - node -c yarn db:reseed"
+      - >
+        docker run
+        --env CLOUDINARY_CLOUD_NAME=$CLOUDINARY_CLOUD_NAME
```

Comment:
You can use pretty much any value for any of these except for the database stuff.  Seeding doesn't use them, and unlike Next.js they don't become a permanent part of the application.

Reply:
I'm assuming we can remove them completely when they are no longer required to be defined.

Reply:
They'll probably always be required for the backend, since it looks for them when it starts up and won't init unless they validate.